### DISCUSSION TITLE
Error on Local Deployment: "Evals available in cloud only" when Adding LLM API Key

### DISCUSSION THREAD

**Discussion by youngsj951104** [langfuse/langfuse#2070] [Support]
Hello,

I encountered an error while trying to integrate an LLM API Key into my local LangFuse setup. Upon entering the API Key, the system responds with an error indicating that evaluations are only available in the cloud. Below is the error message and the associated details I received:
[
    {
        "error": {
            "json": {
                "message": "Evals available in cloud only",
                "code": -32603,
                "data": {
                    "code": "INTERNAL_SERVER_ERROR",
                    "httpStatus": 500,
                    "path": "llmApiKey.all",
                    "zodError": null
                }
            }
        }
    }
]
![image](https://github.com/langfuse/langfuse/assets/76549585/7cb3e1ba-4486-44d2-a55f-4ba16c3f07bc)

From the error, it seems like this operation is restricted to cloud deployments. However, I was under the impression that API Key integration might also be supported locally. Could you please clarify if this is intended behavior, or if there might be a configuration or deployment step that I am missing? Additionally, any guidance on how to resolve this issue or work around it would be greatly appreciated.

Thank you for your assistance.

**Comment by marcklingen** [comment]
Thanks for raising this and sorry for the confusion.

This setting is related to features which are currently in public beta as they require some new infrastructure which will be part of langfuse v3: https://github.com/orgs/langfuse/discussions/1902

I just removed this settings panel to make this less confusing: https://github.com/langfuse/langfuse/releases/tag/v2.43.1
